### PR TITLE
Implement the homepage [DAH-746]

### PR DESCRIPTION
**Pull request by james-wills-sf**
[DAH-746]

Branches off of the bump bloom pr: https://github.com/SFDigitalServices/sf-dahlia-web/pull/1476 so we can use working links.

[DAH-746]: https://sfgovdt.jira.com/browse/DAH-746

### CODE REVIEW

**Comment by akegan** [comment]
@james-wills-sf heads up that there are merge conflicts in `tl.json` because of my right to return PR. My change was that I ran grunt translations and it alphabetized the keys 
image

**Comment by akegan** [comment]
@james-wills-sf can you add review instructions to the ticket when you get a chance?

File: app/javascript/pages/HomePage.tsx [hunk 1]
```diff
@@ -33,14 +34,18 @@ const HomePage = (props: HomePageProps) => {
       </div>
       <Hero
         title={t("welcome.title")}
-        buttonTitle={t("welcome.seeRentalListings")}
+        backgroundImage={getAssetPath("bg@1200.jpg")}
```
Comment (akegan) [inline app/javascript/pages/HomePage.tsx:37]: it looks like for the angular app we might change the resolution of the picture based on the resolution of the device.  Going from bg@400.jpg to bg@2400.jpg. Is that something we should consider here too? Or should we check in with Exygy to see if they intentionally dropped that?

Comment (james-wills-sf) [reply app/javascript/pages/HomePage.tsx:37]: I saw that but didn't haven't put much thought into how we'd do it. It's not supported in bloom, I'm not sure if it was conscious or not but I'm pretty sure they're not doing anything smart on their end to scale it for us.

It looks okay to me at all resolutions, but we could make a ticket to make this responsive? I think we should look into how to get the screen size via react hooks if that's possible

Comment (akegan) [reply app/javascript/pages/HomePage.tsx:37]: might be worth asking Jesse about it to see if it's something on their list. If he thinks it's OK then I'm OK with it too.

**Comment by james-wills-sf** [comment]
@akegan review instructions added and merge conflicts fixed!

File: app/javascript/pages/HomePage.tsx (hunk 1 above)
Comment (akegan) [reply app/javascript/pages/HomePage.tsx:37]: @james-wills-sf it seems like Jesse's answer was fairly non-committal, but they could maybe make an issue in bloom for it? Should we follow up and make sure that ticket gets created?

**Comment by akegan** [comment]
Minor issue from QA: the disclaimer is going to an acgov site: 
image
It should go to `/disclaimer`

File: app/javascript/pages/HomePage.tsx (hunk 1 above)
Comment (james-wills-sf) [reply app/javascript/pages/HomePage.tsx:37]: Yeah I'll make sure an issue gets created (or I'll just create one)

**Comment by james-wills-sf** [comment]
@akegan updated the disclaimer link. I also noticed that the privacy policy link wasn't being translated so I fixed that too.

**Review by akegan — APPROVED** [review]
LGTM thanks for the changes!

File: app/javascript/pages/HomePage.tsx (hunk 1 above)
Comment (james-wills-sf) [reply app/javascript/pages/HomePage.tsx:37]: Issue linked here: https://github.com/bloom-housing/bloom/issues/1303

**Pull request merged**